sdk/resourcemanager/Proto.Client/authorization/Proto.Authorization.csproj

```diff
@@ -23,6 +23,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\Azure.ResourceManager.Core\src\Azure.ResourceManager.Core.csproj" />
     <ProjectReference Include="Azure.ResourceManager.Authorization\Azure.ResourceManager.Authorization.csproj" />
+    <Compile Include="../../../../common/ManagementCoreShared/**/*.cs" Link="Shared/Management/%(RecursiveDir)%(Filename)%(Extension)" />
```

Comment:
don't think this is needed here, can we set IsOldTrack2 false to achieve the same thing?